package.json

```diff
@@ -52,22 +44,34 @@
     "@typescript-eslint/eslint-plugin": "^5.42.1",
     "@typescript-eslint/parser": "^5.42.1",
     "eslint": "^8.40.0",
-    "eslint-config-prettier": "^8.5.0",
+    "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-jest": "^27.1.5",
     "eslint-plugin-jsdoc": "^39.6.2",
     "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^4.2.1",
-    "jest": "^26.4.2",
+    "eslint-plugin-prettier": "^5.2.1",
+    "jest": "^29.7.0",
     "outdent": "^0.8.0",
+    "prettier": "^3.3.3",
+    "prettier-plugin-packagejson": "^2.5.2",
     "rimraf": "^3.0.2",
-    "ts-jest": "^26.5.6",
+    "ts-jest": "^29.2.5",
     "typescript": "~4.8.4"
   },
+  "peerDependencies": {
+    "prettier": "^3.3.3"
+  },
```

Comment:
This forces a consumer to use this specific version range of `prettier` - curious, is there any unwanted outcome of including this in direct dependencies so that this package can use its specific version regardless of the rest of the dep tree?

Reply:
I assume every consumer already has Prettier installed. We could change it to a version range like `>=3.0.0`.